reference/gradle-plugin-configuration.md

```diff
@@ -143,6 +143,21 @@ After the goal finishes executing, run `git diff` (or your VCS system's equivale
 
 ![Example of changes made to netflix conductor by the rewriteRun task](../.gitbook/assets/rewrite-fix-git-diff-output.png)
 
+### JVM args that can be added to the Gradle command line
+
+It is possible to control the active recipe and style using JVM arguments in the command line.
+
+* To change the active recipe, you can specify: `activeRecipe`, `activeRecipes`, `rewrite.activeRecipe`, or `rewrite.activeRecipes` (all of them do the same thing and take the same input).
+* To change the style, you can specify: `activeStyle`, `activeStyles`, `rewrite.activeStyle`, or `rewrite.activeStyles` (all of them do the same thing and take the same input).
```

Comment:
Let's standardize to only use the `rewrite.` ones; we already had a community member propose to standardize in
- https://github.com/openrewrite/rewrite-maven-plugin/pull/712

Reply:
@timtebeek Should I include both `rewrite.activeRecipe` and `rewrite.activeRecipes`?

Reply:
Probably good, yeah, to signify that folks can pass in one or more recipes.